central/package.json

```diff
@@ -10,12 +10,14 @@
     "@material-ui/lab": "^4.0.0-alpha.48",
     "@mui/icons-material": "^5.6.2",
     "@mui/material": "^5.13.3",
+    "@types/lodash": "^4.14.202",
```

Comment:
we're using `debounce` from `lodash` to debounce text changes generating API requests on the search bar